apps/pom.xml

```diff
@@ -166,7 +165,7 @@
         <protoc.version>3.25.1</protoc.version>
         <protobuf.version>3.25.1</protobuf.version>
         <jacoco.version>0.8.11</jacoco.version>
-
+        <jakarta.servlet.version>6.1.0-M2</jakarta.servlet.version>
```

Comment:
As we talked about this morning, I think we want to move this back to 6.0. Other than that, this looks good to me 👍